vulnerabilities/api.py

```diff
@@ -48,11 +48,30 @@ class MinimalPackageSerializer(serializers.HyperlinkedModelSerializer):
     Used for nesting inside vulnerability focused APIs.
     """
 
+    def get_affected_vulnerabilities(self, package):
+        parent_affected_vulnerabilities = package.fixed_package_details.get("vulnerabilities") or []
+        affected_vulnerabilities = []
+
+        for vuln in parent_affected_vulnerabilities:
+            self.get_vulnerability(vuln, affected_vulnerabilities)
```

Comment:
You should not pass the `affected_vulnerabilities` list object to be updated, but instead append the value returned by the `get_vulnerability` function call.